src/pages/Spawn.tsx

```diff
@@ -54,11 +48,8 @@ export const Spawn: React.VFC = () => {
         </SideNavGroup>
       </SideNav>
       <PageWrapper>
-        <Route path={routes.spawnHost} component={SpawnHost} />
-        <Route path={routes.spawnVolume} component={SpawnVolume} />
+        <Outlet />
```

Comment:
I think the nested routes make sense to live in the component they are implemented in.